apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/html/libraries/iframe-edit-mode.css.ts

```diff
@@ -343,6 +343,64 @@ export const getEditPageCss = (timestampId: string): string => {
         z-index: 1 !important;
     }
 
+    ${timestampId} .inline-editing--saving::before {
+        background: rgba(200,200,200, .6);
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        content: "";
+        left: 0;
+    }
+
+    ${timestampId} .inline-editing--error::before {
+        background: rgba(255, 0, 0, 0.2);
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        content: "";
+        left: 0;
+    }
+
+    [data-mode="full"] > * {
+        pointer-events: none;
+    }
+
+    [data-mode="full"].active > * {
+        pointer-events: auto;
+    }
+
+    p.mce-edit-focus, 
+    h1.mce-edit-focus, 
+    h2.mce-edit-focus, 
+    h3.mce-edit-focus, 
+    h4.mce-edit-focus, 
+    h5.mce-edit-focus, 
+    h6.mce-edit-focus {
+        color: black !important;
+    }
+
+    .mce-edit-focus p,
+    .mce-edit-focus h1, 
+    .mce-edit-focus h2, 
+    .mce-edit-focus h3, 
+    .mce-edit-focus h4, 
+    .mce-edit-focus h5, 
+    .mce-edit-focus h6 {
+        color: black !important;
+    }
```

Comment:
Can't we just do here: `.mce-edit-focus` or `*`? Because we can get any tag here.